packages/flutter/lib/src/widgets/container.dart

```diff
@@ -332,10 +375,12 @@ class Container extends StatelessWidget {
   /// The decoration to paint in front of the [child].
   final Decoration? foregroundDecoration;
 
+  /// Constraints for the [child]'s width/height, in logical pixels.
+  final double? width, height;
```

Comment:
Please separate these out into two distinct properties with distinct doc comments to avoid any confusion.